### PR TITLE
show correct percentage in comparison.html if ref-tokens are passed

### PR DESCRIPTION
**Pull request by pagnotta**
fix https://gitlab.fokus.fraunhofer.de/dashcenc/web-platform-tests-wave/issues/54

### CODE REVIEW

**Comment by aldafu** [comment]
WIP, don't merge before format style is adapted (e.g. no ; ' instead of ")

File: tools/wave/src/network/api/results-api-handler.js [hunk 1]
```diff
@@ -169,6 +189,92 @@ class ResultsApiHandler extends ApiHandler {
...
+    for (let i = 0; i < tokenArr.length; i++) {
+      const token = tokenArr[i];
+      sessionResults[token] = {};
+      const sessionResult = await this._resultsManager.getResults(token);
+      for (let api in sessionResult) {
+        sessionResults[token][api] = 0;
+        const apiResult = sessionResult[api];
+        let passedSubTests = 0;
+        for (let result in apiResult) {
+          const subtests = apiResult[result].subtests;
```
Comment (aldafu) [inline tools/wave/src/network/api/results-api-handler.js]: `subtests` can be `undefined` and cause a failed Promise rejection

**Comment by aldafu** [comment]
can be closed. new pull request for this is: https://github.com/cta-wave/WMAS2017/pull/14

**Comment by louaybassbouss** [comment]
ok I will close this PR.

**Pull request closed without merging**